lib/lunchbot_web/live/component/shared_cart/office_selector.ex

```diff
@@ -0,0 +1,12 @@
+defmodule LunchbotWeb.SharedCart.OfficeSelector do
```

Comment:
another nit: For the file structure, I know that there's the `modal.ex` in `live/components`. Since these are all components can we have them be in the same directory? Doesn't matter what the name of the folder is (`components` or `component`)

Reply:
This is actually no longer a component, so I will probably be pulling it out with the rest, but keeping these 4 files in their own `shared_cart` folder